service/src/main/java/uk/gov/hmcts/reform/fpl/service/search/SearchService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.service.search;
+
+import lombok.RequiredArgsConstructor;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.service.ccd.CoreCaseDataService;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Map.of;
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SearchService {
+
+    private final CoreCaseDataService coreCaseDataService;
+
+    public List<CaseDetails> search(String property, LocalDate day) {
```

Comment:
I am thinking we might need to limit this by LA as additionoal param. Maybe Gail would recoommend?

How we could marry it with feature tooggle? I don't think checking one by noe is feasible, but code doesn't know which LAs are enabled :(

Reply:
again, it was agreed with business

Reply:
Tomasz Powroznik  4:06 PM
hi
Just wondering if https://tools.hmcts.net/jira/browse/FPLA-1102 is affected by
https://tools.hmcts.net/jira/browse/FPLA-1527
4:07
if cases for different LA are found should i send all of them to ctsc, or filter out those which are not: 
-Swansea
-Cardiff
-Liverpool (edited) 

Gail Heather:house_with_garden:  4:20 PM
It's OK - we send to CTSC as per 1102
4:21
And they have instructions to forward it to relevant court IF it isn't one of the courts they are looking after

Tomasz Powroznik  4:21 PM
ok, thank you

Gail Heather:house_with_garden:  4:22 PM
I will ask Jay if she wants to do any improvements on it but other courts should be moving across eventually anyway, they just got scared because of the virus but obviously at the moment it's in their interests to move to the centralised service (I think)...
4:22
And we don't want to have to change notifications every time a new court is added - that would be even more of a nightmare than it already is.